docs: added troubleshooting page

hirentimbadiya opened the pull request:
This PR Fixes #1804

mowies commented on docs/content/en/docs/troubleshooting.md:
```diff
@@ -0,0 +1,44 @@
+
+This guide will help you address common issues that you might encounter while using KLT for managing your workloads. Below are some common problems and their solutions:
+
+## KLT is installed but KLT isn't aware of my workloads
```
```suggestion
## KLT is installed but it isn't aware of my workloads
```

mowies commented on docs/content/en/docs/troubleshooting.md:
```diff
@@ -0,0 +1,44 @@
+
+> **_NOTE:_**  This section particularly affects to clusters managed by ArgoCD.
+
+If you've uninstalled Keptn Lifecycle Toolkit and are now facing issues scheduling or deleting pods, follow these steps:
```
```suggestion
If you have uninstalled Keptn Lifecycle Toolkit and are now facing issues scheduling or deleting pods, follow these steps:
```

mowies reviewed (changes requested):
I left a few comments, please also fix the failing markdown linting checks

hirentimbadiya commented on docs/content/en/docs/troubleshooting.md:
```diff
@@ -0,0 +1,44 @@
+# Troubleshooting Common Problems with KLT
```
should i add cascade property ?

hirentimbadiya commented:
@mowies, I have implemented suggested changes.

agardnerIT commented:
@hirentimbadiya  markdownlint is complaining. The issues it wants resolved can be found by clicking the Details link next to the red cross above ([direct link](https://github.com/keptn/lifecycle-toolkit/actions/runs/5799784198/job/15732572149?pr=1860))

You can also try running `make markdownlint-fix` as that can fix some (but not all) common issues.

hirentimbadiya commented:
> Re

I have fixed markdown linting checks, kindly review the PR. @agardnerIT

mowies commented:
hey @hirentimbadiya ! I fixed the last markdown linter errors and changed the page title and weight. I think it's ready for merging now 👍🏼

hirentimbadiya commented:
> hey @hirentimbadiya ! I fixed the last markdown linter errors and changed the page title and weight. I think it's ready for merging now 👍🏼

Thanks @mowies

RealAnna commented on docs/content/en/docs/troubleshooting.md:
```diff
@@ -0,0 +1,58 @@
+
+ArgoCD does not delete various CRDs and webhooks, when uninstalling applications, causing lingering resources.
+
+For cleanup instructions, refer to this [issue](https://github.com/keptn/lifecycle-toolkit/issues/1828).
```
Hey @hirentimbadiya thanks for the contrib! I like that you link here so people can read more about it, but I would say here it makes sense also to copy over the instructions from the issue so that we can keep them updated in case something changes.

hirentimbadiya replied:
thank you @RealAnna and  I'll implement your suggested changes

hirentimbadiya commented:
@RealAnna , could you please review the PR? I've included the cleanup instructions. Additionally, kindly let me know if there are any suggested changes.

agardnerIT commented:
@hirentimbadiya I'm so sorry but the strict markdown checks are still failing: https://github.com/keptn/lifecycle-toolkit/actions/runs/5812269050/job/15767638037?pr=1860

@aepfli markdownlint causing more pain.

hirentimbadiya commented:
> @hirentimbadiya I'm so sorry but the strict markdown checks are still failing: https://github.com/keptn/lifecycle-toolkit/actions/runs/5812269050/job/15767638037?pr=1860
> 
> @aepfli markdownlint causing more pain.

I'll solve those

hirentimbadiya commented:
@agardnerIT, I have solved some markdownlint checks, kindly review PR.

mowies commented on docs/content/en/docs/troubleshooting.md:
```diff
@@ -0,0 +1,71 @@
+installed and configured on the cluster.
+
+[The OpenTelemetry observability page](https://lifecycle.keptn.sh/docs/implementing/otel/)
+contains more information on how to configure this.
```
```suggestion
contains more information on how to configure this.

```

RealAnna commented:
Thank you @hirentimbadiya we will merge this 🚀 😸
